run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# We are using socat in order ant API to be accessible outside of docker
+# container.
+socat tcp-listen:9980,reuseaddr,fork tcp:localhost:10980 &
```

Comment:
We don't need `socat`. I need to run some tests and remove it from the official images as well. You can just run the ant farm on its own. Unless you have a reason that I don't know about, of course.

Reply:
Without `socat` I was not able to connect to the Ant Farm's renter inside docker.

I think it's because the renter API address is `127.0.0.1:10980`, where `127.0.0.1` is the docker container's localhost, not the host machine's localhost. The connection from the host machine targets docker container's IP (not docker localhost) and nothing is served on that IP.

Anyway I can see if you can get rid of `socat` in Sia image and if it's viable also for Ant Farm.

Reply:
Oh, so you actually need. Good, then let it be.